src/main/resources/ixia-ipfix.json

```diff
@@ -0,0 +1,245 @@
+{
+  "enterprise_number": 3054,
```

Comment:
NB: Just curious , how was this json file generated ?

Reply:
Ah, great catch! This isnt meant to be here, its mean to be a configurable value. See above where i mention infoElementDefs in IpfixCode. Will delete.